sql/wsrep_client_service.h

```diff
@@ -48,8 +48,27 @@ class Wsrep_client_service : public wsrep::client_service
   void emergency_shutdown()
   { throw wsrep::not_implemented_error(); }
   void will_replay();
+  void signal_replayed();
   enum wsrep::provider::status replay();
+  enum wsrep::provider::status replay_unordered()
+  {
+    throw wsrep::not_implemented_error();
```

Comment:
I'd prefer avoid throwing exceptions over architectural boundaries. DBUG_ASSERT(0) and returning suitable error code should be sufficient.